CONTRIBUTING.md

```diff
@@ -1,14 +1,14 @@
 Contributing to PrestaShop
 --------------------------
 
-PrestaShop is an open-source e-commerce solution. Everyone is welcome and even encouraged to contribute with their own improvements.
+PrestaShop is an open source ecommerce solution. Everyone is welcome and even encouraged to contribute with their own improvements.
```

Comment:
It's the same for e-commerce. I see no reason to change the working here.
Could you please revert your changes.
The only thing acceptable in this pull request is what you did with links :)

Reply:
I reverted my changes :+1: 
Thank you @PierreRambaud